cli/configssh.go

```diff
@@ -524,7 +525,7 @@ func (r *RootCmd) configSSH() *serpent.Command {
 			}
 
 			if !bytes.Equal(configRaw, configModified) {
-				err = writeWithTempFileAndMove(sshConfigFile, bytes.NewReader(configModified))
+				err = atomic.WriteFile(sshConfigFile, bytes.NewReader(configModified))
```

Comment:
Does this new module do much more than what we were already doing or solve an issue with the previous approach?

Reply:
The new module also copies the file mode from the previous file, and does an `fsync` to avoid renaming a zero-length file - they're about the same aside from that.